src/index.js

```diff
@@ -74,23 +73,28 @@ const renderTodos = () => {
       });
     }
 
-    todos.forEach((todo,index) => {
+    clearBtn.addEventListener('click', () => {
+      todos.splice(0, todos.length);
```

Comment:
- [Optional] The splice() method is a mutating method. This is why we should use the Array.filter() method as requested.